docs/pages/get-started/expo-go.mdx

```diff
@@ -45,25 +47,54 @@ The manifest is similar to an **index.html** on the web. It is served from the r
 
-We release a new SDK version approximately every quarter. Find out which [versions of React Native map to which versions of the Expo SDK](/versions/latest/#each-expo-sdk-version-depends-on-a-react-native-version).
+<Collapsible summary="What if I open a project with an unsupported SDK version?">
 
-## Login into Expo Go
+When running a project that was created for an unsupported SDK version in Expo Go, you'll see the following error:
+```
+"Project is incompatible with this version of Expo Go"
+```
+To fix this, upgrading your project to a [supported SDK version](/versions/latest/#each-expo-sdk-version-depends-on-a-react-native-version) is highly recommended. If you want to learn how to do it, see [Upgrade the project to a new SDK Version](#upgrade-the-project-to-a-new-sdk-version).
 
-Open the Expo Go app after it has finished installing. If you have [created an account with Expo CLI](/more/expo-cli/#authentication), you can sign in by clicking the **Login** button in the top header on the **Home** tab. Signing in will make it easier for you to open projects in the Expo Go app while developing them &mdash; they will appear automatically under the **Projects** section on the **Home** tab of the app.
+</Collapsible>
 
-> Sometimes, it can be useful to run your app directly on your computer instead of on a separate physical device. If you would like to set this up, you can learn more about [installing an Android Emulator](/workflow/android-studio-emulator) and [installing the iOS Simulator (macOS only)](/workflow/ios-simulator).
+<Collapsible summary="How do I upgrade my project from an unsupported SDK version?">
+
+Refer to the ["Upgrading Expo SDK" guide](/workflow/upgrading-expo-sdk-walkthrough.mdx) for instructions for upgrading to a specific SDK version.
 
-## Production
+</Collapsible>
 
-Expo Go is not intended for public distribution. It is a development client that is not optimized for performance. We offer a suite of tools for building and submitting your apps directly to the app stores. For more information, see [Distribution](/distribution/introduction).
+<Collapsible summary="How can I install an older version of Expo Go to run my project?">
 
-You can create your own Expo Go type app by using the [`expo-dev-client`](/develop/development-builds/introduction/) package. This package allows you to create a native client with any custom native code or configuration or a third-party library that requires configuring native code. For more information, see [development builds](/develop/development-builds/introduction/).
+If you can't upgrade your project to a newer SDK version, you can still run it using an older version of Expo Go — however, this method will not work for physical iOS devices because it is not possible to install apps directly (sideload). If you need to run your project on a physical iOS device, consider creating a [development build](/develop/development-builds/introduction.mdx).
```

Comment:
```suggestion
If you can't upgrade your project to a newer SDK version, you can still run it using an older version of Expo Go. However, this method will not work for physical iOS devices because it is not possible to install apps directly (sideload). If you need to run your project on a physical iOS device, consider creating a [development build](/develop/development-builds/introduction).
```

- Let's break the sentence here.